build.sbt

```diff
@@ -443,6 +443,19 @@ lazy val `guide-guide` =
     Dependencies.guideJsDeps,
     Some((`guide-backend`, "io.udash.web.styles.GuideCssRenderer")),
     Def.task(Some((`guide-commons` / sourceDirectory).value / "main" / "assets"))
+  ).settings(
+    Compile / copyAssets := {
```

Comment:
I'd rather you add this logic to the existing `copyAssets` logic in `frontendExecutable` - FontAwesome is available on both frontends anyway via `guide-commons`.